content/en/integrations/guide/source-code-integration.md

```diff
@@ -147,3 +156,4 @@ In the [Continuous Profiler][2], you can directly access traces in the source re
 [2]: https://app.datadoghq.com/account/settings#integrations/github-apps
 [3]: https://app.datadoghq.com/apm/error-tracking
 [4]: https://docs.github.com/en/developers/apps/getting-started-with-apps/about-apps
+[5]: https://docs.datadoghq.com/tracing/profiler/search_profiles/
```

Comment:
`Continuous Profiler` was wrongly linking to gut GitHub apps integration tile. I didn't find a valid link in history.  
I suggest linking here if we want to keep a link.